### PR TITLE
revert photo upload timeout

### PR DESCRIPTION
**Pull request by adammeyer**

### CODE REVIEW

**Comment by adammeyer** [comment]
@ndedios FYI

File: src/app/designationEditor/photoModal/photo.modal.js [hunk 1]
```diff
@@ -1,28 +1,39 @@
...
     this.photos = photos;
   }
 
-  uploadComplete(response) {
-    this.photos = response.data;
-    this.uploading = false;
+  uploadComplete() {
+    //refresh photos (set timeout to give Adobe time to add to DAM)
+    this.$timeout(() => {
+      this.designationEditorService.getPhotos(this.designationNumber, this.campaignPage).then((response) => {
```
Comment (ndedios) [inline src/app/designationEditor/photoModal/photo.modal.js]: @adammeyer is this method sending the campaign code to the endpoint? image upload and image listing (POST and GET) does not process the campaign parameter (as we previously check yesterday, images are shared between a designation and their campaigns)

Comment (adammeyer) [inline src/app/designationEditor/photoModal/photo.modal.js]: correct, I can remove it

Comment (adammeyer) [inline src/app/designationEditor/photoModal/photo.modal.js]: All images are under /content/dam/give/designations. Regardless if they're a designation or a campaign.  Is that correct?

Comment (ndedios) [inline src/app/designationEditor/photoModal/photo.modal.js]: Yes, always are in the same location.

**Pull request merged**